ARCs/arc-0069.md

```diff
@@ -31,9 +31,9 @@ The ASA parameters should follow the following conventions:
 * *Asset Name* (`an`): no restriction.
 * *Asset URL* (`au`): a URI pointing to digital media file. This URI:
     * **SHOULD** be persistent.
-    * **SHOULD** link to a file small enough to fetch quickly in a gallery view.
+    * **SHOULD** link to a file small enough to fetch quickly in a gallery view for files different from 3D Mesh. For 3D Mesh files it is **RECOMMENDED** that a client application downloads and renders them.
     * **MUST** follow [RFC-3986](https://www.ietf.org/rfc/rfc3986.txt) and **MUST NOT** contain any whitespace character.
-    * **SHOULD** specify media type with `#` fragment identifier at end of URL. This format **MUST** follow: `#i` for images, `#v` for videos, `#a` for audio, `#p` for PDF, or `#h` for HTML/interactive digital media.  If unspecified, assume Image.
+    * **SHOULD** specify media type with `#` fragment identifier at end of URL. This format **MUST** follow: `#i` for images, `#v` for videos, `#a` for audio, `#p` for PDF, `#h` for HTML/interactive digital media, or `#m` for 3D Mesh format.  If unspecified, assume Image.

```

Comment:
Adding #m seems simple enough. Would limiting the kinds of meshes make sense though? We would want meshes a browser could render effectively

Reply:
Rendering in browsers would be easy  using the GLB format (https://www.khronos.org/gltf/) which is widely used. 
However, there are other renderers besides browsers. We must not lose the possibility of using other  clients and formats such as USDZ (https://graphics.pixar.com/usd/release/spec_usdz.html) or 3d engines not based on webGL.

Reply:
I've had good experiences with https://modelviewer.dev/, which supports glTF and GLB

Reply:
Perfect! This does not exclude the use of other standard formats already in use and supported by other platforms.